src/components/atoms/OcSearchBar/OcSearchBar.vue

```diff
@@ -5,33 +5,44 @@
     class="oc-search"
     :class="{ 'oc-search-small': small }"
   >
-    <div class="oc-width-expand oc-position-relative">
-      <span v-if="icon" class="oc-search-icon" @click="focusSearchInput">
-        <oc-icon v-show="!loading" :name="icon" fill-type="line" />
-        <oc-spinner
-          v-show="loading"
-          :size="spinnerSize"
-          :aria-label="loadingAccessibleLabelValue"
+    <div class="oc-width-expand">
+      <div class="oc-width-1-1 oc-position-relative">
+        <span v-if="icon" class="oc-search-icon" @click="focusSearchInput">
+          <oc-icon v-show="!loading" :name="icon" fill-type="line" />
+          <oc-spinner
+            v-show="loading"
+            :size="spinnerSize"
+            :aria-label="loadingAccessibleLabelValue"
+          />
+        </span>
+        <input
+          ref="searchInput"
+          :class="inputClass"
+          :aria-label="label"
+          :value="searchQuery"
+          :disabled="loading"
+          :placeholder="placeholder"
+          @input="onType($event.target.value)"
+          @keydown.enter="onSearch"
         />
-      </span>
-      <input
-        ref="searchInput"
-        :class="inputClass"
-        :aria-label="label"
-        :value="searchQuery"
-        :disabled="loading"
-        :placeholder="placeholder"
-        @input="onType($event.target.value)"
-        @keydown.enter="onSearch"
-      />
+        <oc-button
+          v-if="query.length > 0"
```

Comment:
is > 0 required?

Reply:
Not necessarily, I didn't touch it - just increased the indentation. I will adjust it if more changed are needed, otherwise let's leave it

Reply:
fair enough